network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -960,6 +968,12 @@ def get_primary_related_articles(self) -> models.QuerySet['BuyersGuideArticlePag
     def get_secondary_related_articles(self) -> models.QuerySet['BuyersGuideArticlePage']:
         return self.get_related_articles()[3:]
 
+    def check_for_categories_featuring_cta(self):
```

Comment:
**Question** Does this method really only do a "check" as the name suggests?

Reply:
Per our discussion, updated to "get_featured_cta"